dotcom-rendering/src/web/components/RichLink.tsx

```diff
@@ -282,21 +282,21 @@ export const RichLink = ({
 									{mainContributor}
 								</div>
 							)}
-							{starRating && starRating > 0 && (
+							{!!starRating && (
```

Comment:
This is a fix, as star ratings of `0` [are actually valid](http://content.guardianapis.com/search?api-key=test&page-size=100&show-fields=starRating&star-rating=0)!